web/profiles/joinup/joinup.info.yml

```diff
@@ -78,8 +98,11 @@ config_devel:
   - og.og_membership_type.og_membership_type
   - og.settings
   - system.theme
+  - user.flood
+  - user.mail
   - user.role.administrator
   - user.role.anonymous
   - user.role.authenticated
   - user.role.moderator
   - user.role.user
```

Comment:
What is the use of the "user" role since we are using authenticated for this purpose?